examples/testnet_simple_signing.py

```diff
@@ -113,10 +126,11 @@ def validate_responses_with_cohort_eth_multisig(
 
 
 def print_signing_result(
-    original_data: bytes, signature_responses: List[SignatureResponse]
+    user_op: UserOperation, signature_responses: List[SignatureResponse]
 ):
     print("\n-----")
-    print(f"Original Message: {original_data}")
+    print(f"UserOperation sender: {user_op.sender}")
+    print(f"UserOperation nonce: {user_op.nonce}")
```

Comment:
Nitpick for output to line up 😅 
```suggestion
    print(f"UserOperation nonce : {user_op.nonce}")
```

Reply:
✔️